cmb2-conditionals.php

```diff
@@ -12,13 +12,13 @@
  *
  * @wordpress-plugin
  * Plugin Name:       CMB2 Conditionals
- * Plugin URI:        https://github.com/jcchavezs/cmb2-conditionals
+ * Plugin URI:        https://github.com/yaronguez/cmb2-conditionals
```

Comment:
I performed that commit after making the pull request in order to use this fork on my own site in the meantime. I assumed it wouldn't be added to the request. I'll move it to a different branch instead.